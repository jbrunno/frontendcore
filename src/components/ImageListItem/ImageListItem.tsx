import React from 'react';
import {
  ImageListItem as MuiImageListItem,
  ImageListItemProps as MuiImageListItemProps,
} from '@mui/material';

export function ImageListItem(props: MuiImageListItemProps) {
  return <MuiImageListItem {...props} />;
}

export default ImageListItem;
