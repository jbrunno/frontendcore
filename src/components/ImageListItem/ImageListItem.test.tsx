import React from 'react';
import { render } from '@test-utils/component-testing';
import { ImageListItem } from './ImageListItem';

test('renders the ImageListItem', () => {
  const { container } = render(<ImageListItem />);
  expect(container).toMatchSnapshot();
});
