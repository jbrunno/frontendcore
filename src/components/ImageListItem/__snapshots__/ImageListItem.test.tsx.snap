// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the ImageListItem 1`] = `
.emotion-0 {
  display: block;
  position: relative;
}

.emotion-0 .MuiImageListItem-img {
  object-fit: cover;
  width: 100%;
  height: 100%;
  display: block;
}

<div>
  <li
    class="MuiImageListItem-root emotion-0"
    style="height: auto; grid-column-end: span 1; grid-row-end: span 1;"
  />
</div>
`;
